website/docs/api/curated-transformer.mdx

```diff
@@ -0,0 +1,546 @@
+| `vocab`             | The shared vocabulary. ~~Vocab~~                                                                                                                                                                                                                   |
+| `model`             | One of the supported pre-trained transformer models. ~~Model~~                                                                                                                                                                                     |
+| _keyword-only_      |                                                                                                                                                                                                                                                    |
+| `name`              | The component instance name. ~~str~~                                                                                                                                                                                                               |
+| `frozen`            | If `True`, the model's weights are frozen and no backpropagation is performed. ~~bool~~                                                                                                                                                            |
+| `all_layer_outputs` | If `True`, the model returns the outputs of all the layers. Otherwise, only the output of the last layer is returned. This must be set to `True` if any of the pipe's downstream listeners require the outputs of all transformer layers. ~~bool~~ |
+
+## CuratedTransformer.\_\_call\_\_ {id="call",tag="method"}
+
+Apply the pipe to one document. The document is modified in place, and returned.
+This usually happens under the hood when the `nlp` object is called on a text
+and all pipeline components are applied to the `Doc` in order. Both
+[`__call__`](/api/transformer#call) and [`pipe`](/api/transformer#pipe) delegate
+to the [`predict`](/api/transformer#predict) and
+[`set_annotations`](/api/transformer#set_annotations) methods.
+
+> #### Example
+>
+> ```python
+> doc = nlp("This is a sentence.")
+> trf = nlp.add_pipe("curated_transformer")
+> # This usually happens under the hood
+> processed = transformer(doc)
+> ```
+
+| Name        | Description                      |
+| ----------- | -------------------------------- |
+| `doc`       | The document to process. ~~Doc~~ |
+| **RETURNS** | The processed document. ~~Doc~~  |
+
+## CuratedTransformer.pipe {id="pipe",tag="method"}
+
+Apply the pipe to a stream of documents. This usually happens under the hood
+when the `nlp` object is called on a text and all pipeline components are
+applied to the `Doc` in order. Both [`__call__`](/api/transformer#call) and
+[`pipe`](/api/transformer#pipe) delegate to the
+[`predict`](/api/transformer#predict) and
+[`set_annotations`](/api/transformer#set_annotations) methods.
```

Comment:
Same comment about the `api` links, best to double-check throughout.

Reply:
Fixed.